pcsx2/gui/AppMain.cpp

```diff
@@ -542,11 +542,13 @@ void Pcsx2App::LogicalVsync()
 	FpsManager.DoFrame();
 	
 	if (EmuConfig.Gamefixes.FMVinSoftwareHack || g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
+		AspectRatioType& art = g_Conf->GSWindow.AspectRatio;
+		if (g_Conf->GSWindow.IsToggleAspectRatioSwitch && !EnableFMV && !FMVstarted && art == AspectRatio_4_3 && prevAR != art)
+			prevAR = art;
```

Comment:
```
If(prevAR != art)
    prevAR = art;
```
is the same as 
```
prevAR = art;
```

But do you really need this? It is only active while not within FMV. But after FMVenable=true it will automatically be overwritten in L553 or is set due to L76.